doc/quickstart/install.rst

```diff
@@ -196,10 +196,46 @@ To install from source for development, follow these instructions.
 -  Test that your installation was successful by running
    ``esmvaltool -h``.
 
-Pre-installed versions on HPC clusters
```

Comment:
Why are you replacing the HPC module load instructions with conda lock instead of just adding?

Reply:
darn, I re-added it, sorries